applications/SeismicApplication/seismic_application.cpp

```diff
@@ -0,0 +1,74 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Mahmoud Zidan
+//
+
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+
+#include "seismic_application.h"
+#include "seismic_application_variables.h"
+#include "includes/variables.h"
+
+#include "geometries/line_gauss_lobatto_3d_2.h"
+
+
+namespace Kratos {
+
+KratosSeismicApplication::KratosSeismicApplication()
+    : KratosApplication("SeismicApplication"),
+    // Adding the fiber beam-column element
+    mFiberBeamColumnElement3D2N(0, Element::GeometryType::Pointer(new LineGaussLobatto3D2<Node<3>>(Element::GeometryType::PointsArrayType(2))))
+    {}
+
+void KratosSeismicApplication::Register()
+{
+    // calling base class register to register Kratos components
+    KratosApplication::Register();
+
+    KRATOS_INFO("") << "    KRATOS  ____       _               _      \n";
```

Comment:
We decided to avoid this, or at least doing it in only one line

Reply:
Ok